task_12/src/css/desktop/header.css

```diff
@@ -0,0 +1,41 @@
+header {
+  display: grid;
+  width: 100%;
+  background-image: url(../img/header-bg.jpg);
+  font-size: 15px;
+  align-items: center;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr)); }
+
+.timeBlock {
+  margin-left: 60px;
+  cursor: pointer; }
+  .timeBlock:hover {
+    transition: 500ms linear;
+    color: $hover-text-color; }
+
+.logInBlock {
+  margin-right: 60px;
+  cursor: pointer;
+  justify-self: end; }
+  .logInBlock:hover {
+    transition: 500ms linear;
+    color: $hover-text-color; }
+
+.logo {
+  display: block;
+  cursor: pointer;
+  justify-self: center; }
+
+.logoImage {
+  width: 200px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  opacity: 1; }
+  .logoImage:hover {
```

Comment:
same here

Reply:
<img width="424" alt="screen shot 2018-02-05 at 15 28 37" src="https://user-images.githubusercontent.com/24464487/35807068-48badb3e-0a8a-11e8-8fc6-084ba6dfe080.png">
Tell me what I'm doing wrong?

Reply:
Sorry about it. We just missed that's compiled file.

However, for the future to avoid such stupid mistakes I suggest you push to the branch only source files, and everything compiled push to `gh-pages`. We will be more careful too.

Reply:
Everything is fine, I probably should also immediately more clearly write that this comment is not in the right file.

Not sure about the division, but still thanks for the idea.